ludwig/models/trainer.py

```diff
@@ -970,11 +971,19 @@ def train(self, training_set, validation_set=None, test_set=None, save_path="mod
                     self.is_coordinator(),
                 )
 
+                early_stopping_steps = final_steps_per_checkpoint * self.early_stop
+
                 if self.is_coordinator():
                     logger.info(
                         f"Training for {total_steps} step(s), approximately "
                         f"{int(total_steps / batcher.steps_per_epoch)} epoch(s)."
                     )
+                    logger.info(
+                        f"Early stopping policy: {self.early_stop} round(s) of evaluation, or {early_stopping_steps} "
+                        f"step(s), which is approximately {int(early_stopping_steps / batcher.steps_per_epoch)} "
```

Comment:
nit: 
```suggestion
                        f"step(s), approximately {int(early_stopping_steps / batcher.steps_per_epoch)} "
```
to be consistent with total_steps info message above